std_msgs/String state
---
std_msgs/Bool status